.gitignore

```diff
@@ -8,5 +8,9 @@ target/
 
 api/tartget/
 omod/target/
+<<<<<<< HEAD
+api/bin/pom.xml 
```

Comment:
here you committed Git merge conflicts, you need to clear these conflicts definitely